src/test/java/api/loans/EndPatronActionSessionTests.java

```diff
@@ -1,5 +1,31 @@
 package api.loans;
 
+import api.support.APITests;
```

Comment:
According to the [provisional import ordering standard](https://issues.folio.org/browse/FOLIO-1667) static imports should go first. Please can this be updated to reflect that standard.

Reply:
I'm using IntelliJ and it appears upgrading to the 2019.3 release candidate has reset my import settings for projects :-/

This might have caused this issue for you as well.

Reply:
yes, you are right, the same problem... 
fixed